apps/masterbots.ai/components/routes/subscription/plans.tsx

```
@@ -156,27 +188,31 @@ export function Plans({ next, goTo }: PlansPros) {
-            plans?.filter(plan => plan.active).sort((a, b) => a.created - b.created).map(plan => (
-              <PlanCard
-                key={plan.id}
-                selectedPlan={selectedPlan}
-                handlePlanChange={handlePlanChange}
-                plan={plan}
-              />
-            ))
-          )}
-          {(loadingPlans && !plans) && (
+          {plans &&
+            plans.length &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)